src/app/compressed-air-assessment/compressed-air-banner/compressed-air-banner.component.ts

```diff
@@ -165,4 +168,25 @@ export class CompressedAirBannerComponent implements OnInit {
     this.emailMeasurDataService.emailItemType.next('CompressedAir');
     this.coreService.showShareDataModal.next(true);
   }
+
+  hideTooltip(btnType: string) {
```

Comment:
For any PRs delete comments used during development, with the exception being PRs that explain why something was done or have domain information